views/partials/social_media_sharing_buttons.hbs

```diff
@@ -1,13 +1,13 @@
 <ul class="social-media-sharing-buttons">
   <li>
-    <a href="https://www.facebook.com/sharer/sharer.php?u={{ SERVER_URL}}" rel="noopener noreferrer" data-analytics-event="facebook" class="sendGA">
+    <a href="https://www.facebook.com/sharer/sharer.php?u={{ SERVER_URL }}" target="_blank" rel="noopener noreferrer" data-analytics-event="facebook" class="sendGA">
     <img class="primary-social-icon facebook-button" alt="Share Firefox Monitor on Facebook" src="/img/social_media_icons/facebook.svg" />
     <img class="secondary-social-icon" src="/img/social_media_icons/facebook-round.svg" alt="Share Firefox Monitor on Facebook" />
     <span>Share</span>
     </a>
   </li>
   <li>
-    <a href={{ encodeURI "Have your accounts been leaked or stolen in a data breach? Find out at Firefox Monitor. Search our database and sign up for alerts." }} rel="noopener noreferrer" data-analytics-event="twitter" class="twitter-share-button sendGA">
+    <a href="https://twitter.com/intent/tweet?url={{ SERVER_URL }}" target="_blank" rel="noopener noreferrer" data-analytics-event="twitter" class="twitter-share-button sendGA">
```

Comment:
For fellow travellers, here's what it looks like in https://cards-dev.twitter.com/validator : 

![twitter-card](https://user-images.githubusercontent.com/557895/44862891-85eb5400-ac30-11e8-9ee9-90349cbac1de.png)

---

And here's what it looks like in the Twitter intent page https://twitter.com/intent/tweet?url=https://fx-breach-alerts.herokuapp.com

![twitter-intent](https://user-images.githubusercontent.com/557895/44862949-ae734e00-ac30-11e8-8ab1-2b98cc98aed1.png)